docs/dps-api.md

```diff
@@ -5,47 +5,92 @@
 1. [Table of Contents](#table-of-contents)
 2. [Endpoints](#endpoints)
 3. [Types](#types)
-   1. [GetFirstRequest](#getfirstrequest)
-   2. [GetFirstResponse](#getfirstresponse)
-   3. [GetLastRequest](#getlastrequest)
-   4. [GetLastResponse](#getlastresponse)
-   5. [GetHeaderRequest](#getheaderrequest)
-   6. [GetHeaderResponse](#getheaderresponse)
-   7. [GetCommitRequest](#getcommitrequest)
-   8. [GetCommitResponse](#getcommitresponse)
-   9. [GetEventsRequest](#geteventsrequest)
-   10. [GetEventsResponse](#geteventsresponse)
-   11. [GetRegistersRequest](#getregistersrequest)
-   12. [GetRegistersResponse](#getregistersresponse)
+    - [GetCommitRequest](#getcommitrequest)
+    - [GetCommitResponse](#getcommitresponse)
+    - [GetEventsRequest](#geteventsrequest)
+    - [GetEventsResponse](#geteventsresponse)
+    - [GetFirstRequest](#getfirstrequest)
+    - [GetFirstResponse](#getfirstresponse)
+    - [GetHeaderRequest](#getheaderrequest)
+    - [GetHeaderResponse](#getheaderresponse)
+    - [GetHeightRequest](#GetHeightRequest)
+    - [GetHeightResponse](#GetHeightResponse)
+    - [GetLastRequest](#getlastrequest)
+    - [GetLastResponse](#getlastresponse)
+    - [GetRegistersRequest](#getregistersrequest)
+    - [GetRegistersResponse](#getregistersresponse)
+    - [GetTransactionRequest](#GetTransactionRequest)
+    - [GetTransactionResponse](#GetTransactionResponse)
+    - [ListCollectionsForBlockRequest](#ListCollectionsForBlockRequest)
+    - [ListCollectionsForBlockResponse](#ListCollectionsForBlockResponse)
+    - [ListTransactionsForBlockRequest](#ListTransactionsForBlockRequest)
+    - [ListTransactionsForBlockResponse](#ListTransactionsForBlockResponse)
+    - [ListTransactionsForCollectionRequest](#ListTransactionsForCollectionRequest)
+    - [ListTransactionsForCollectionResponse](#ListTransactionsForCollectionResponse)
```

Comment:
I'm not sure why you changed this, but my extension will probably automatically change this back :D

Reply:
I changed it because it's a pain in the ass to add/remove/move elements from a numbered list (at least in my IDE :p) but I'm fine with keeping the numbers.